polaris-react/src/components/Button/Button.scss

```diff
@@ -424,8 +424,8 @@
     position: relative;
     border-color: currentColor;
     // stylelint-disable selector-max-class
-    box-shadow: 0 0 0 var(--p-border-width-1) currentColor;
-    @include focus-ring($border-width: var(--p-border-width-2));
+    box-shadow: 0 0 0 1px currentColor;
```

Comment:
For the sake of this PR can we use the border token and update it in the future?